### PR TITLE
Add role for sylius plus Rbac

### PR DESCRIPTION
**Pull request by ghost**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?   | yes/no
| Fixed issue | #...

### CODE REVIEW

File: src/Resources/config/routes/admin/customer.yaml [hunk 1]
```diff
@@ -2,8 +2,18 @@ synolia_sylius_gdpr_admin_anonymize_customer:
...
 
 synolia_sylius_gdpr_admin_export_customer_datas:
     path: /customers/{id}/export-data
     methods: POST|GET
     controller: Synolia\SyliusGDPRPlugin\Controller\ExportDataController
+    defaults:
+        _sylius:
+            permission: true
+            section: admin
+            alias: plugin_synolia_gdpr_export__customer_data
```
Comment (GalloisLuca) [inline src/Resources/config/routes/admin/customer.yaml]: There is too much underscore plugin_synolia_gdpr_export_**_**customer_data

File: src/Resources/translations/messages.en.yaml [hunk 2]
```diff
@@ -6,4 +6,11 @@ sylius:
...
                     anonymize: Anonymize
                     export_data: Export data
-                    gdpr_title: GDPR
\ No newline at end of file
+                    gdpr_title: GDPR
+sylius_plus:
+    rbac:
+        parent:
+            plugin_synolia_gdpr_anonymize_customer: GDPR Anonymize
+            plugin_synolia_gdpr_export__customer_data: GDPR Export data
```
Comment (GalloisLuca) [inline src/Resources/translations/messages.en.yaml]: Same

File: src/Resources/translations/messages.fr.yaml [hunk 3]
```diff
@@ -6,4 +6,11 @@ sylius:
...
                     anonymize: Anonymizer
                     export_data: Récupération des informations
-                    gdpr_title: RGPD
\ No newline at end of file
+                    gdpr_title: RGPD
+sylius_plus:
+    rbac:
+        parent:
+            plugin_synolia_gdpr_anonymize_customers: RGPD Anonymizer
+            plugin_synolia_gdpr_export__customer_datas: RGPD Récupération des informations
```
Comment (GalloisLuca) [inline src/Resources/translations/messages.fr.yaml]: same

File: src/Resources/translations/messages.fr.yaml [hunk 4]
```diff
@@ -6,4 +6,11 @@ sylius:
...
+        synolia_sylius_gdpr_admin_anonymize_customer: Anonymizer
```
Comment (GalloisLuca) [inline src/Resources/translations/messages.fr.yaml]: Anonymiser

File: src/Resources/translations/messages.fr.yaml [hunk 5]
```diff
@@ -6,4 +6,11 @@ sylius:
                     not_found: Client introuvable
                     anonymize: Anonymizer
                     export_data: Récupération des informations
-                    gdpr_title: RGPD
\ No newline at end of file
+                    gdpr_title: RGPD
+sylius_plus:
+    rbac:
+        parent:
+            plugin_synolia_gdpr_anonymize_customers: RGPD Anonymizer
```
Comment (GalloisLuca) [inline src/Resources/translations/messages.fr.yaml]: RGPD Anonymisation

**Review by GalloisLuca — CHANGES_REQUESTED** [review]
I asked for some changes, re-request me for review after you update the PR ! :)

**Comment by GalloisLuca** [comment]
Can you rename your commit plz ?

**Review by GalloisLuca — APPROVED** [review]

**Comment by GalloisLuca** [comment]
Seems good to me, we are waiting for @oallain to merge :stuck_out_tongue:

**Review by oallain — APPROVED** [review]

**Comment by oallain** [comment]
Thanks @tuyetrinhvo and @GalloisLuca :rocket:

**Pull request merged**